engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/RangePredicateFromContext.java

```diff
@@ -16,19 +18,44 @@
 	 * Require at least one of the targeted fields to be "lower than" the given value,
 	 * in addition to being "higher than" the value provided to the
 	 * former <code>{@link RangePredicateFieldSetContext#from(Object) from}</code> call.
+	 * <p>
+	 * This method will apply DSL converters to {@code value} before Hibernate Search attempts to interpret it as a field value.
+	 * See {@link DslConverter#ENABLED}.
 	 *
 	 * @param value The upper bound of the range. May be null, in which case the range has no upper bound,
 	 * but this is only possible if the lower bound ({@link RangePredicateFieldSetContext#from(Object)})
 	 * was not null.
 	 * The signature of this method defines this parameter as an {@link Object},
 	 * but a specific type is expected depending on the targeted field.
-	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-parametertype">there</a> for more information.
+	 * See {@link DslConverter#ENABLED} for more information.
+	 *
+	 * Upper bound is included by default in the range.
+	 *
+	 * @return A context allowing to exclude the upper bound from the range, to set options or to get the resulting predicate.
+	 */
+	default RangePredicateLimitTerminalContext to(Object value) {
+		return to( value, DslConverter.ENABLED );
+	}
+
+	/**
+	 * Require at least one of the targeted fields to be "lower than" the given value,
+	 * in addition to being "higher than" the value provided to the
+	 * former <code>{@link RangePredicateFieldSetContext#from(Object) from}</code> call.
+	 *
+	 * @param value The upper bound of the range. May be null, in which case the range has no upper bound,
+	 * but this is only possible if the lower bound ({@link RangePredicateFieldSetContext#from(Object)})
+	 * was not null.
+	 * The signature of this method defines this parameter as an {@link Object},
+	 * but a specific type is expected depending on the targeted field and on the {@code dslConverter} parameter.
+	 * See {@link DslConverter} for more information.
+	 * @param dslConverter Controls how the {@code value} should be converted before Hibernate Search attempts to interpret it as a field value.
+	 * See {@link DslConverter} for more information.
```

Comment:
Same here. I would avoid to repeat this line two times, one for each parameter.
I would move it to the end of the javadoc using a `@see`.